content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,448 @@
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+     </tbody>
+ </table>
+ 
+ ### Resource data matrix
+ 
+ <table>
+     <thead>
+         <tr>
+             <th>Resource</th>
+             <th>Attribute</th>
+             <th>Additional information</th>
+         </tr>
+     </thead>
+     <tbody>
+         <tr>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               <tr>
+             <td rowspan=9>Pool</td>
+             <td>id</td>
+             <td></td>
+         </tr>
+         <tr>
+             <td>name</td>
+             <td></td>
+         </tr>
+         <tr>
+             <td>storage_id</td>
+             <td></td>
+         </tr>
+         <tr>
+            <td>native_storage_pool_id</td>
+            <td></td>
+         </tr>
+         <tr>
+            <td>description</td>
+            <td></td>
+         </tr>
+         <tr>
+            <td>status</td>
+            <td></td>
+         </tr>
+         <tr>
+            <td>total_capacity</td>
+            <td></td>
+         </tr>
+         <tr>
+            <td>used_capacity</td>
+            <td></td>
+         </tr>
+         <tr>
+            <td>free_capacity</td>
+            <td></td>
```

Comment:
what do each of the supported attributes mean?
There is no information provided to the user.

Reply:
IMO @Shruthi-1MN We can provide description and other additional info in 'Additional info' tab. like used capacity in GB

Reply:
True, we can keep it as "Description" which can always contain fill some useful info (meaning, or unit, special note etc)

Reply:
done